### ISSUE TITLE
ActionUpdateThread.OLD_EDT deprecated

### ISSUE THREAD

**Issue by rivancic** [jfrog/jfrog-idea-plugin#472]
**Describe the bug**
During the scan process IntelliJ throws following exception and breaks the scan.

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.jfrog.ide.idea.actions.ScanTimeLabelAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: org.jfrog.idea]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:200)
```

**To Reproduce**

IntelliJ IDEA version: IntelliJ IDEA 2024.1.1 (Ultimate Edition)
JFrog plugin version: 2.6.7
Operating system: Windows 10 Enterprise

1) Login with SSO to JFrog server

2) Start the scan

3) It is scanning project dependencies
![image](https://github.com/user-attachments/assets/5a57a49f-4e7c-4eb5-b181-dec1b3589e28)

4) It fails in the last step
![image](https://github.com/user-attachments/assets/31461bd5-d1d0-44ff-8fb4-bdffff315d71)

**Expected behavior**
Dependencies should be analyzed and report displayed

**Additional context**

**Comment by eyalk007** [comment]
Hi @rivancic,
Thank you for reporting the bug!
It has been solved and will be released in the next cycle

**Comment by rivancic** [comment]
Any info on the next release? It's been a while since the last one in January.

**Comment by eyalk007** [comment]
@rivancic 
The next release is planned for next week 😁

**Comment by eyalk007** [comment]
Solved as of version `2.7.0`
@rivancic
